app/helpers/assessment_query_helper.rb

```diff
@@ -34,7 +34,7 @@ def sort(assessments, order, direction)
       assessments = assessments.order(created_at: dir)
     else
       # Verify this is a sort request for a symptom column.
-      symptom_columns = Assessment.get_unique_symptoms_for_assessments(assessments.pluck(:id)).pluck(:name)
```

Comment:
Why not improve the performance of the `Assessment.get_unique_symptoms_for_assessments` method so that it can be reused elsewhere?

Reply:
great question, so this method was originally being used in 2 places and I started out by optimizing it for both, but the more I continued to optimize it, it seemed like it would be better to just optimize them individually because they were being used for different purposes/returning different fields